graphql-dgs-client/build.gradle.kts

```diff
@@ -40,3 +40,7 @@ dependencies {
     testImplementation(project(":graphql-dgs-spring-graphql-starter"))
     testImplementation(project(":graphql-dgs-spring-graphql-starter-test"))
 }
+
```

Comment:
Why is this needed for only some but not all modules? For e.g. why don;t we need this for `graphql-dgs-pagination` or `graphql-dgs-spring-boot-micrometer`

Reply:
I added module-info to micrometer and pagination as well now. I tried extended validation as well, but there's a split package between `graphql-java` and `graphql-java-extended-validation`, so that's not working.

For Jars that are not real modules you can load them as automatic-modules on the module path, or just load them from the classpath.